src/metaArrow.h

```diff
@@ -72,8 +72,8 @@ class METAIO_EXPORT MetaArrow : public MetaObject
     void  Length(float length);
     float Length(void) const;
 
-    void  Lenght(float length) {this->Length(length);}
-    float Lenght(void) const {return Length();}
+    void  Length(float length) {this->Length(length);}
+    float Length(void) const {return Length();}
```

Comment:
please review this specific change.

Reply:
It looks like this was left for backwards compatibility and should be retained.